tools/ctl/ide/commands/commands.go

```diff
@@ -329,6 +328,23 @@ func NewRebuildGroupCmds(app *tview.Application, rb *rebuilder.Rebuilder, modalF
 						{Text: "Please summarize this rebuild failure in one sentence."},
 						{Text: logs},
 					}
+					if strings.Contains(in.Message, "content mismatch") {
```

Comment:
do we include the logs in the context if the failure is a diff? I guess it could maybe help suggest an alternative but I'm less sure it would improve the performance of the clustering. input tokens are cheap, i guess....